_layouts/category.html

```diff
@@ -38,9 +38,9 @@
                         <div ng-if="category.title === currentLocation">
                             <div class="title-container page-header">
                                 <img ng-src="{{ category.icon.path }}" style="width: 125px; height: 125px" alt="{{ category.icon.alt }}" title="{{ category.icon.alt }}" />
-                                <div class="title-description">
+                                <div class="title-description" style="width: 700px">
                                     <h3>{{ category.title }}</h3>
-                                    <p>{{ category.description }}</p>
+                                    <p>{{ category.description2 }}</p>
```

Comment:
Rather than duplicating all the values in the json below (to make what I see as a "short" and a "long" description, it might make sense to use some of the Jekyll conditional logic. I haven't tested this, but I think you should be able to do something like this:

```markdown
{% if category.long_description %}
  {{ category.long_description }}
{% else %}
  {{ category.description }}
{% endif %}
```

Then you only need to add a `long_description` to the Radius category in the json file, and not need to add to all the others.

Here are some references that might help if you run in to issues:

- https://jekyllrb.com/docs/liquid/
- https://idratherbewriting.com/documentation-theme-jekyll/mydoc_conditional_logic.html
- https://stackoverflow.com/a/41646032

Reply:
@IanLee1521 Please review the latest commits on this PR. I could not get the conditional to work w/o also displaying the logic in the {% %} and/or w/o ugly line breaks or vertical spacing problems. Also could not get *only* the long description to display if the long description existed. I tried all combos of if/elsif/else/unless. I tried "hiding" the {% %} logic in <p class> and similar markup, as is done on header.html where menu dropdowns are determined.

So with these new commits, this feature is not in fact conditional:
1. Short description displays for categories that only have short, with no discernable line break or spacing issues.
2. Both descriptions display if both exist. This means the RADIUSS long description has a colon & space at the beginning so it looks like it belongs after the short description and displays OK (in the attached screen shot, the red line shows where short ends and long begins). This would become problematic if we decide to use the long description in another scenario.

I feel like this is an ugly solution and possibly worse than `description` vs `description2`. The time spent on this seems disproportionate to the payoff. What do you think?
![radiuss description](https://user-images.githubusercontent.com/31740953/104363210-bbbfc900-54c9-11eb-8c4b-e60161ab1ca0.PNG)

Reply:
It's something about the way that `{% raw %}` messes with the syntax, but we need that to make the rest of the html to look OK... Not sure if you're happy with this otherwise, if yes, I think it's fine to mere.